tools/c7n_gcp/c7n_gcp/resources/logging.py

```diff
@@ -23,15 +23,129 @@ class LogSink(QueryResourceManager):
+        scope_key = 'parent'
+        scope_template = 'projects/{}'
+        id = 'name'
+
+        @staticmethod
+        def get(client, resource_info):
+            return client.execute_query('get', {
+                'sinkName': 'projects/{project_id}/sinks/{name}'.format(
+                    **resource_info)})
+
+
+@resources.register('log-project-metric')
+class LogProjectMetric(QueryResourceManager):
+
+    class resource_type(TypeInfo):
+        service = 'logging'
+        version = 'v2'
+        component = 'projects.metrics'
+        enum_spec = ('list', 'metrics[]', None)
+        scope_key = 'parent'
+        scope_template = 'projects/{}'
+        id = 'name'
+
+        @staticmethod
+        def get(client, resource_info):
+            return client.execute_query('get', {
+                'metricName': 'projects/{project_id}/metrics/{name}'.format(
+                    **resource_info)})
+
+
+@resources.register('log-project')
+class LogProject(QueryResourceManager):
+
+    class resource_type(TypeInfo):
+        service = 'logging'
+        version = 'v2'
+        component = 'projects.logs'
+        enum_spec = ('list', 'logNames[]', None)
+        scope_key = 'parent'
+        scope_template = 'projects/{}'
+        id = 'project_id'
+
+    def augment(self, resources):
+        out_resources = []
+        for resource in resources:
+            resource_type, project_id, key, log_id = resource.split('/')
+            out_resources.append({
+                'logName ': resource,
+                'resource_type': resource_type,
+                'project_id': project_id,
+                'key': key,
+                'log_id': log_id
+            })
+        return out_resources
+
+
+@resources.register('log-project-exclusion')
+class LogProjectExclusion(QueryResourceManager):
+
+    class resource_type(TypeInfo):
+        service = 'logging'
+        version = 'v2'
+        component = 'projects.exclusions'
+        enum_spec = ('list', 'exclusions[]', None)
+        scope_key = 'parent'
+        scope_template = 'projects/{}'
+        id = 'name'
+
+        @staticmethod
+        def get(client, resource_info):
+            return client.execute_query('get', {
+                'name': 'projects/{project_id}/exclusions/{name}'.format(
+                    **resource_info)})
+
+
+@resources.register('log-exclusion')
+class LogExclusion(QueryResourceManager):
+
+    class resource_type(TypeInfo):
+        service = 'logging'
+        version = 'v2'
+        component = 'exclusions'
+        enum_spec = ('list', 'exclusions[]', None)
+        scope_key = 'parent'
+        scope_template = 'projects/{}'
+        id = 'name'
+
+        @staticmethod
+        def get(client, resource_info):
+            return client.execute_query('get', {
+                'name': '{type}/{project_id}/exclusions/{name}'.format(
+                    **resource_info)})
+
+
+@resources.register('log-monitored-resource-descriptor')
```

Comment:
this shouldn't be a resource

Reply:
its read only metadata..

Reply:
@jonathana, @jondavidi - What do think about it? Should we remove this resource or not?

Reply:
you should remove it, it serves no purpose.